lib/project.ts

```diff
@@ -630,6 +630,18 @@ export class Project implements Project.IProject {
 		}).future<void>()();
 	}
 
+	public isTypeScriptProject(): IFuture<boolean> {
```

Comment:
It is in our hooks - typescript-compilation.ts
I'll reuse it there and keep the code here.